config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml

```diff
@@ -1,5 +1,18 @@
 presubmits:
   kubernetes-sigs/kernel-module-management:
+  - name: pull-kernel-module-management-check-commits-number
+    always_run: true
+    decorate: true
+    optional: true
+    annotations:
+      testgrid-dashboards: sig-node-kernel-module-management
+      testgrid-tab-name: check-commits-number
+      description: "make sure each PR to kernel-module-management has a single commit."
+    spec:
+      containers:
+      - image: golang:1.18
```

Comment:
Do we need Go for this job? This could be a simpler image, such as `bitnami/git:latest`.

Reply:
Fixed.